### ISSUE TITLE
photo albom - crashing client

### ISSUE THREAD

**Issue by kitovoi111** [mortuusars/Exposure#49]
[latest.log](https://github.com/mortuusars/Exposure/files/14356819/latest.log)

**Comment by mortuusars** [comment]
I don't see any errors related to exposure in this log. Please provide a crash-report.

**Comment by kkylern** [comment]
Yeah, the thing is that that the photo album is crashing game without any logs. Game is freezing up and the only way to close it is to use a task manager.

**Comment by mortuusars** [comment]
@kkylern 
Any additional information? Versions, environment, steps to reproduce the issue?

**Comment by kkylern** [comment]
Sorry for not giving any details at first. Now I was testing at the same time as I was writing so I couldn't answer immediately.
- Forge 4.3.30 1.19.2 
- Singleplayer and multiplayer 

Just opening empty album (simple or empty signed) is causing this issue. I thought it was because of heavy modpack (259 mods). 
So I have tried this mod and 20 other mods and problem had occured.
And then I have tried this mod standalone and the problem didn't occured.

And that is what led to me to test mods one by one. So I find out that I can't find out anything! Even though the photo album was crashing game several times while I was testing, It suddenly stopped. Why? I don't know but I need to dig further. I will keep you updated if I actually find something.
But yeah, it's probably some incompatibility. Heavy modpack is still crashing.

**Comment by kkylern** [comment]
Alright, I found a way to cause actual crash and I have got a crash report on my hands. I have placed the album on lectern and opened it.
[crash-2024-02-07_13.14.20-client.txt](https://github.com/mortuusars/Exposure/files/14466843/crash-2024-02-07_13.14.20-client.txt)
I did this on heavy modpack but I tested and it is happening on small instance too, where the album started working normal. Maybe it's a different crash?
[crash-2024-03-02_04.25.05-client.txt](https://github.com/mortuusars/Exposure/files/14466859/crash-2024-03-02_04.25.05-client.txt)
